segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -983,6 +988,11 @@ private Void handleException(long requestId, String segment, long offset, String
         return null;
     }
 
+    private boolean errorCodeExists(Throwable e) {
+        val errorCode = WireCommands.ErrorMessage.ErrorCode.valueOf(e.getClass());
+        return (errorCode == WireCommands.ErrorMessage.ErrorCode.UNSPECIFIED) ? false : true;
```

Comment:
You don't need `? :` here. Change `==` to `!=` and return the result.

Reply:
🤦.. rookie mistake.